website/src/get-started/wordcount-example.md

```diff
@@ -739,6 +755,12 @@ $ mvn package -Pnemo-runner && java -cp target/word-count-beam-bundled-0.1.jar o
      --runner=NemoRunner --inputFile=`pwd`/pom.xml --output=counts
 ```
 
+{:.runner-jet}
+```
+$ mvn package -P jet-runner && java -cp target/word-count-beam-bundled-0.1.jar org.apache.beam.examples.DebuggingWordCount \
+     --runner=JetRunner --jetLocalMode=3 --inputFile=$pwd/pom.xml --output=counts
```

Comment:
```suggestion
     --runner=JetRunner --jetLocalMode=3 --output=counts
```
It looks like most of the runners are running against the default inputFile (the Shakespeare corpus) which demonstrates the PAsserts succeeding in the example.

Nemo is running against `pom.xml` which demonstrates failing the PAssert in the example.

I suggest following the majority!

Reply:
Good point, thanks.